docs/tutorials/heatmaps.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<p>Although not stated anywhere I don't think, AFAIK all docs only capitalize the first word in each section title. So "Heatmap Configurations" -&gt; "Heatmap configurations" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
done for all sections.